processor/coralogixprocessor/config.go

```diff
@@ -0,0 +1,36 @@
+package coralogixprocessor // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/coralogixprocessor"
+
+import "fmt"
+
+type cacheConfig struct {
+	maxCacheSizeMebibytes int64 `mapstructure:"max_cache_size_mib"`
```

Comment:
```suggestion
	maxCacheSizeMib int64 `mapstructure:"max_cache_size_mib"`
```
If you want the name to be a bit more concise.